src/clojure/nrepl/core.clj

```diff
@@ -66,7 +66,9 @@
 (defn- delimited-transport-seq
   [client termination-statuses delimited-slots]
   (with-meta
-    (comp (partial take-until (comp #(seq (clojure.set/intersection % termination-statuses))
+    (comp (partial take-until (comp #(seq (clojure.set/intersection
+                                           (keyworded-set %)
```

Comment:
Why is this change needed?

Reply:
tbh, not sure. This is from one of @dotemacs's commits.

Reply:
Well, I think you should check if this actually breaks something as it looks weird to me.

Reply:
Reverted. Everything seems to still work.